jrpc/jrpcClient.go

```diff
@@ -26,13 +26,27 @@ import (
 	"io"
 	"net/http"
 	"time"
+
+	"fortio.org/fortio/version"
+)
+
+// Client side and common code.
+
+const (
+	UserAgentHeader = "User-Agent"
 )
 
 // Default timeout for Call.
 var timeout = 60 * time.Second
 
+// UserAgent is the User-Agent header used by client calls (also used in fhttp/).
+var UserAgent = "fortio.org/fortio-" + version.Short()
+
 // SetCallTimeout changes the timeout for further Call calls, returns
-// the previous value (default in 60s).
+// the previous value (default in 60s). Value is used when a timeout
```

Comment:
```suggestion
// The previous value (default in 60s). Value is used when a timeout
```

Reply:
this is continuation of same sentence "returns the previous value" so it would not work with "returns The previous value:

Reply:
I don't see weird/odd things. I have to figure out a use for it (do let me know) to battle test it.

Reply:
the pager duty call we're making for instance - I'll make a few MRs once this is released and we can iterate if it's not great